docs/source/widgets.rst

```diff
@@ -501,7 +503,8 @@ Supported platforms: GNU/Linux.
   and ``${sign}`` (signal level, in dBm)
 
 .. _strftime(3): https://linux.die.net/man/3/strftime
+.. _Google Security page: https://myaccount.google.com/security?gar=1
```

Comment:
From [the instruction I found], it seems that the app password thing is hidden behind the *Signing in to Google* thing.  Is that still the case?  If so you might want to point to that instruction instead (I also assume that link will be valid even if Google change the procedure in the future).